src/domains/citizen-auth-common/03_storage.tf

```diff
@@ -48,6 +48,27 @@ resource "azurerm_private_endpoint" "lollipop_assertion_storage_blob" {
   tags = var.tags
 }
 
+resource "azurerm_private_endpoint" "lollipop_assertion_storage_queue" {
+  name                = "${module.lollipop_assertions_storage.name}-queue-endpoint"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.data_rg.name
+  subnet_id           = data.azurerm_subnet.private_endpoints_subnet.id
+
+  private_service_connection {
+    name                           = "${module.lollipop_assertions_storage.name}-queue"
+    private_connection_resource_id = module.lollipop_assertions_storage.id
+    is_manual_connection           = false
+    subresource_names              = ["queue"]
```

Comment:
LGTM, just a question: is possible to list more resources there insted of creating more private endpoints? 
Something like
subresource_names = ["queue","blob]

Reply:
no :(